.github/workflows/ci.yaml

```diff
@@ -636,3 +640,116 @@ jobs:
         asset_path: cartographer-conventions-samples-bundle.tar
         asset_name: cartographer-conventions-samples-bundle-${{ steps.get_version.outputs.VERSION }}.tar
         asset_content_type: application/x-tar
+    - uses: vmware-tanzu/carvel-setup-action@v1
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+    - name: Install crane
+      run: |
+        cd $(mktemp -d -t kind.XXXX)
+        curl -L https://github.com/google/go-containerregistry/releases/download/v0.8.0/go-containerregistry_Linux_x86_64.tar.gz | tar -xz
+        chmod +x ./crane
+        sudo mv ./crane /usr/local/bin
+        cd -
+    - name: Login to GitHub Container Registry
+      uses: docker/login-action@v1
+      with:
+        registry: ghcr.io
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+    - name: Relocate bundle to public registry
+      run: |
+        set -o errexit
+        set -o nounset
+        set -o pipefail
+
+        version=${{ steps.get_version.outputs.VERSION }}
+
+        imgpkg copy --tar cartographer-conventions-bundle.tar --to-repo "ghcr.io/${{ github.repository }}/bundle"
+        crane tag "ghcr.io/${{ github.repository }}/bundle" "${version}"
```

Comment:
oh, neat! 

should we do this on `ghcr.io/${{ github.repository }}/bundle@$digest` instead? so that we're not relying on the latest push? pretty improbable there'd be an actual race here but .. tagging a digest sounds more right (nitpicking from my part, not a big deal)

Reply:
getting the digest has the same race, so it's a wash.

What I'd really like is the ability to `imgpkg copy` to a tag, alas...